index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="em">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="container">
+            <div class="box box1">Usama Masood</div>
+            <div class="box box2"><img src="Enabled.png" alt=""></div>
+            
+        </div>
```

Comment:
- [ ] This should be the toolbar of your page and you should use proper HTML tags to wrap the elements for better semantics and increase the SEO of your page. Kindly wrap this section in a `header` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

 - [ ] The logo should be on the left and has a link (`a` tag). Kindly fix that. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

 - [ ] The hamburger button is on the right and has a `nav` tag around it. The nav tag is for semantics reasons and the right position for convention and user experience for mobile devices. Kindly change that.